autoload/slime/targets/conemu.vim

```diff
@@ -9,11 +9,8 @@ function! slime#targets#conemu#config() abort
 endfunction
 
 function! slime#targets#conemu#send(config, text)
-  " use the selection register to send text to ConEmu using the windows clipboard (see help gui-clipboard)
-  " save the current selection to restore it after send
-  let tmp = @*
-  let @* = a:text
-  call slime#common#system("conemuc -guimacro:%s print", [a:config["HWND"]])
-  let @* = tmp
+  " Use the selection register to send text to ConEmu using the slime paste file
+  let paste_file = slime#common#write_paste_file(a:text)
+  call slime#common#system("conemuc -guimacro:%s pastefile 2 %s", [a:config["HWND"], paste_file])
```

Comment:
all in all, I'm surprised that using the pastefile would be _faster_ than the clipboard … 🤔 

---

> The paste file has the S flag set, which tells vim not to bother calling fsync() on the file
…
> The file doesn't contain anything that needs to persist across reboots.

That wasn't my understanding of `fsync` — doesn't that create a race-condition if we
1. write (but skip flush)
2. turn around and read the file

?

---

another concern is that the pastefile sticks around, holding on to your last slime send — I'm not sure everybody feels comfortable with that, so I've been avoiding it when possible (only `screen` was still using it, because there were no other choice 😮‍💨 )